src/runtime_src/core/tools/xbmgmt2/flash/xmc.cpp

```diff
@@ -35,6 +35,22 @@
 namespace XBU = XBUtilities;
 #include "boost/format.hpp"
 
+static std::map<int, std::string> scStatusMap = {
+        {0, "NOT READY"},
+        {1, "READY"},
+        {2, "BSL_UNSYNCED"},
+        {3, "BSL_SYNCED"},
+        {4, "BSL_SYNCED_SC_NOT_UPGRADABLE"},
+        {5, "READY_SC_NOT_UPGRADABLE"},
+ 
+};
+
+static std::map<int, std::string> cmcStatusMap = {
+        {0, "NOT READY"},
+        {1, "READY"},
+        {2, "STOPPED"},
+        {4, "PAUSED"},
+};
```

Comment:
Factorize the code.
I am reading this twice... :-)

Reply:
yes, xbmgmt2 is a dup of xbmgmt

Reply:
And? Twice the code, twice the bugs?...

Reply:
This is by design, the xbmgmt2 will be replacing xbmgmt in near future release. @AShivangi can explain this.